### DISCUSSION TITLE
Backend Tech Stack

### DISCUSSION THREAD

**Discussion by OrdepA17** [uprm-inso4101-2023-2024-S1/semester-project-tutoring-app#68] [Polls]
As mentioned in the previous discussion, there are multiple options to create and work with the database.  Here you can select the option you think would be best for developing.

**Comment by print-Eruki** [comment]
Another option can be:

Javascript, MySQL, Amazon RDS (https://aws.amazon.com/rds/mysql/)

Due to the app already being in js, mysql is pretty standard and simple, and there are a lot of examples on how to setup the amazon rds

Feel free to add into as an option, if it is viable.

**Comment by OrdepA17** [comment]
As discussed, the voting shall now be closed and the tech stack to be used for the back end shall be Python, SQLite, and Supabase. The assignees have already created the initial structure of the database with some mock data so that the assignees of the query selection can test their work.
